Homework_4/4s_test_1.py

```diff
@@ -0,0 +1,13 @@
+# Вычислить число c заданной точностью d
+# Пример:
+# при d = 0.001, π = 3.142 10^(-1) ≤ d ≤10^(-10)
+
+N = (float(input("Input number: ")))
+d = (int(input("Input d: ")))
```

Comment:
```suggestion
N = float(input("Input number: "))
d = int(input("Input d: "))
```
А почему у вас d целочисленный тип? Ведь в задании говорится, что оно меньше единицы. Даже диапазоны указаны, насколько оно меньше единицы.